src/templates/functions.jsx

```diff
@@ -2,16 +2,96 @@ import React from 'react';
 
 import { html as toHtml } from '../indices-to-html';
 
+/**
+ * Adds markup to some common text patterns
+ *
+ *  - Bullet lists
+ *  - Todo lists, WP style
+ *  - Inline `code` snippets
+ *  - Header: description/explanation paragraphs
+ *
+ * Note: This code is only meant to serve until a
+ * proper parser can be built up to convert the
+ * unstructured text into structured data. Since at
+ * this time we still create HTML strings directly
+ * and on every render this function will serve
+ * sufficiently but it should not be looked upon
+ * as good example code!
+ *
+ * @param {String} text input list of blocks
+ * @returns {String} marked-up text
+ */
+const toBlocks = text =>
```

Comment:
Is text already stripped of html by this point? 

Overall this looks ok so far for a quick and dirty replace. I'd recommend adding unit tests for these cases so refactoring is pretty easy.